packages/xo-web/src/common/intl/messages.js

```diff
@@ -205,6 +205,9 @@ const messages = {
   formReset: 'Reset',
   formSave: 'Save',
   formNotes: 'Notes',
+  formStatus: 'Status',
+  formConnected: 'Connected',
+  formDisconnected: 'Disconnected',
```

Comment:
`form*` entries are reserved for generic form entries.

Do you think this qualifies?

Reply:
I can rename the entries' label to `statusLabel, connectedLabel, ...`.The idea is to mutualise theses labels given that they are used in many views.

Reply:
Fine by me.